README.md

```diff
@@ -200,9 +208,9 @@ Here's an example of invoking Platform9 Express against a number of hosts:
 .
 .
 ```
-Here's an example of invoking Platform9 Express a single host groups and performing role deployments (based on metadata defined in /opt/pf9-express/inventory/hosts):
+Here's an example of invoking Platform9 Express a single host groups (host groups are either "pmo" for Openstack and "pmk" for Kubernetes), performing role deployments (based on metadata defined in /opt/pf9-express/inventory/hosts) and registering them automatically to the control plane
```

Comment:
Change "Platform9 Express a single host groups" to "Platform9 Express against a single host group"

Change "control plane" to "management plane"

Reply:
corrected